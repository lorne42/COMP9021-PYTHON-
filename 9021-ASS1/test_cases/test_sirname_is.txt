I have met Sirs Alice, Bob and Carol this morning. "
Sir Bob is a Knave," Sir Carol said.
Sir Alice replied: "At least one of Sir Bob and Sir Carol is a Knight." Should I
trust them?