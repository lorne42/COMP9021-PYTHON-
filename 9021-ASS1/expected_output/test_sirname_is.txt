The Sirs are: Alice Bob Carol
There are 2 solutions.